projects/capt/research.html

```diff
@@ -10,6 +10,11 @@ <h2><a name="highlights"></a>  Research Highlights </h2>
  
 <ul>
 
+<!--- 2019  ---->
+<h4> 2019</h4> 
+
+<li>Eyring, V. and coauthors, 2019: Taking Earth system model evaluation to the next level</li> 
```

Comment:
Should this have a link to the article?

Reply:
Good catch @mauzey1 it's findable at https://doi.org/10.1038/s41558-018-0355-y - was published a week ago

Reply:
@durack1 Should I merge this PR and add the link afterwards?

Reply:
@mauzey1 whatever is easier for you, this could be pulled as a branch across to your local repo, edited, then submitted - whatever is easier for you